lib/ansible/modules/cloud/google/gcdns_zone.py

```diff
@@ -54,6 +54,12 @@
               create a TLD and will fail.
         required: true
         aliases: ['name']
+    zone_id:
+        description:
+            - Google Cloud DNS zone ID.
+            - Used when a new zone created. If absent, ID is generated from domain name.
+        required: false
+        default: null
```

Comment:
This needs a version_added key, per Shippable. `ERROR: lib/ansible/modules/cloud/google/gcdns_zone.py:0:0: E309 version_added for new option (zone_id) should be 2.4. Currently 0.0 (75%)`